bin/runtime/src/lib.rs

```diff
@@ -568,10 +568,10 @@ impl pallet_staking::Config for Runtime {
     type BenchmarkingConfig = StakingBenchmarkingConfig;
     type WeightInfo = PayoutStakersDecreasedWeightInfo;
     type CurrencyBalance = Balance;
-    type OnStakerSlash = NominationPools;
     type HistoryDepth = HistoryDepth;
     type TargetList = pallet_staking::UseValidatorsMap<Self>;
     type AdminOrigin = EnsureRoot<AccountId>;
+    type EventListeners = NominationPools;
```

Comment:
Same as above, what this one is intended for?

Reply:
No functional changes, see https://github.com/paritytech/substrate/pull/14527 for more info